### PR TITLE
[WebGL2] Fuse avtivation and accelerate 1x1 conv

### PR DESCRIPTION
**Pull request by ganwenyao**

### CODE REVIEW

**Comment by ganwenyao** [comment]
fixed.

File: src/nn/webgl2/Model.js [hunk 1]
```diff
@@ -56,9 +56,9 @@ export default class Model {
       let inputIndex = inputs[0].index;
       let outputBuffer = outputs[0].buffer;
       let outputIndex = outputs[0].index;
-      let operationStart = performance.now();
+      // let operationStart = performance.now();
```
Comment (huningxin) [inline src/nn/webgl2/Model.js]: could you please remove this line?

File: src/nn/webgl2/Model.js [hunk 2]
```diff
@@ -56,9 +56,9 @@ export default class Model {
...
       this._layers.forEach((layer, i) => {
-        let start = performance.now();
+        // let start = performance.now();
```
Comment (huningxin) [inline src/nn/webgl2/Model.js]: ditto

**Comment by huningxin** [comment]
@ganwenyao , I see your latest push has mobilenet_ssd code. It would be better to include them in another PR. Let's focus on fuse and 1x1 in this PR.

**Comment by huningxin** [comment]
I am OK to leave the debugging code in comment. Going to merge.

**Pull request merged**